### DISCUSSION TITLE
Error after deploying dashboard to railway

### DISCUSSION THREAD

**Discussion by MoathCodes** [mirumee/saleor-dashboard#2907] [Q&A]
Hi,
I tried deploying dashboard app to railway and I setup the API_URI and did some configurations using the saleor docs but the moment I open the instance **I see the email and password fields but I can't login and this error pops up on the right side**:

`
Something went wrong
Unexpected token '<', "<html> <h"... is not valid JSON
Unexpected token '<', "<html>
<h"... is not valid JSON
`
Am I missing a variable or a configuration?

Thanks for your time.

**Comment by peelar** [comment]
Hello @IronDragonDev 👋 

Did you set up all the env vars described in [`configuration.md`](https://github.com/saleor/saleor-dashboard/blob/main/docs/configuration.md)?

**Comment by karolkielecki** [comment]
Hi @IronDragonDev, that error happens when API_URI doesn't have a slash at the end. E.g., `https://demo.saleor.io`, but it should be `https://demo.saleor.io/`. Could you please try again?
